CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ This file is used to list changes made in each version of the nginx cookbook.
 
 ## Unreleased
 
+- Add seperate properties for nginx process user/group (Issue #572)
```

Comment:
Please add a note here that you're deprecating the other property and also mention which resource you're changing here.

Reply:
Done